test/integration/nickserver_test.rb

```diff
@@ -40,15 +40,13 @@ def test_GET_served_via_SKS
     end
   end
 
-  def test_POST_served_via_SKS
```

Comment:
why remove test for requests via POST? I like supporting either GET or POST, depending on the preference of the client.